Projects/UI/HGDesignSystem/Sources/Extensions/View+.swift

```diff
@@ -8,6 +8,61 @@
 import SwiftUI
 
 public extension View {
+    /// Modifier 분기 적용
+    @ViewBuilder
+    func `if`<Content: View>(_ condition: Bool, content: (Self) -> Content) -> some View {
+        if condition {
+            content(self)
+        } else {
+            self
+        }
+    }
     
+    /// Radius 설정
+    func setRadius(_ radius: CGFloat, corners: [CornerType] = CornerType.allCases) -> some View {
+        self.clipShape(
+            .rect(
+                topLeadingRadius: corners.contains(.topLeft) ? radius : .zero,
+                bottomLeadingRadius: corners.contains(.bottomLeft) ? radius : .zero,
+                bottomTrailingRadius: corners.contains(.bottomRight) ? radius : .zero,
+                topTrailingRadius: corners.contains(.topRight) ? radius : .zero
+            )
+        )
+    }
+    
+    /// Set rounded border
+    @ViewBuilder func roundedBorder(
+        _ color: Color,
+        radius: CGFloat = 8,
+        linewidth: CGFloat = 1
+    ) -> some View {
+        self
+            .clipShape(RoundedRectangle(cornerRadius: radius))
+            .overlay {
+                RoundedRectangle(cornerRadius: radius)
+                    .stroke(color, lineWidth: linewidth)
+            }
+    }
```

Comment:
```suggestion
        self
            .setRadius(radius)
            .overlay {
                RoundedRectangle(cornerRadius: radius)
                    .stroke(color, lineWidth: linewidth)
            }
    }
```
로 대체해도 괜찮아보여서 제안해봅니당

Reply:
반영하겠습니당 🫡